soc/nordic/nrf54h/Kconfig

```diff
@@ -67,3 +69,36 @@ config SOC_NRF54H20_CPUFLPR
 	depends on RISCV_CORE_NORDIC_VPR
 
 rsource "gpd/Kconfig"
+
+config SOC_NRF54H20_GLOBAL_HSFLL_RESTRICT_MIN_FREQ
+	bool "Restrict minimum global HSFLL clock frequency"
+	select NRFS
+	select NRFS_GDFS_SERVICE_ENABLED
+	select CLOCK_CONTROL
+	default y if SOC_NRF54H20_CPUAPP
```

Comment:
Why this is enabled by default when APP core is selected? What if there is an application that uses Radio core only?

Reply:
This has been removed :)